data/countries/SG.yaml

```diff
@@ -0,0 +1,130 @@
+# @attrib https://en.wikipedia.org/wiki/Public_holidays_in_Singapore
+
+holidays:
+  SG:
+    names:
+      en: Singapore
+    langs:
+      - en
+    zones:
+      - Asia/Singapore
+    dayoff: sunday
+    days:
+      01-01:
+        _name: 01-01
+      substitutes 01-01 if sunday then next monday:
+        substitute: true
+        _name: 01-01
+      chinese 01-0-01:
+        name:
+          en: Chinese New Year
+      chinese 01-0-02:
+        name:
+          en: Chinese New Year
+      substitutes chinese 01-0-01 if sunday then next tuesday:
+        substitute: true
+        name:
+          en: Chinese New Year
+      substitutes chinese 01-0-02 if sunday then next monday:
+        substitute: true
+        name:
+          en: Chinese New Year
+      easter -2:
+        _name: easter -2
+      05-01:
+        _name: 05-01
+      substitutes 05-01 if sunday then next monday:
+        substitute: true
+        _name: 05-01
+      08-09:
+        name:
+          en: National Day
+      substitutes 08-09 if sunday then next monday:
+        substitute: true
+        name:
+          en: National Day
+      12-25:
+        _name: 12-25
+      substitutes 12-25 if sunday then next monday:
+        substitute: true
+        _name: 12-25
+      1 Shawwal:
+        _name: 1 Shawwal
+        name:
+          en: Hari Raya Puasa
+      10 Dhu al-Hijjah:
+        _name: 10 Dhu al-Hijjah
+        name:
+          en: Hari Raya Haji
+      # https://en.wikipedia.org/wiki/Vesak
+      '2001-05-07':
```

Comment:
I like this pragmatic approach for now.

Reply:
well, it will work until we miss the yearly updates.

Reply:
Yea, I need to take the time to work through the calendar and determine the pattern.